arbstate/daprovider/util.go

```diff
@@ -34,6 +34,10 @@ type DASWriter interface {
 	fmt.Stringer
 }
 
+type DASKeysetFetcher interface {
+	GetByHash(context.Context, common.Hash) ([]byte, error)
```

Comment:
better if that's renamed.. GetKeysetByHash?

will add to clarity and prevent using the wrong fetcher for wrong purpose

Reply:
Fixed, thanks!